packages/console/src/hooks/use-swr-fetcher.ts

```diff
@@ -1,18 +1,35 @@
-import { RequestErrorBody } from '@logto/schemas';
+import { ArbitraryObject, RequestErrorBody } from '@logto/schemas';
 import { HTTPError } from 'ky';
 import { useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
 import { BareFetcher } from 'swr';
 
 import useApi, { RequestError } from './use-api';
 
 const useSwrFetcher = () => {
   const api = useApi({ hideErrorToast: true });
+  const { t } = useTranslation();
   const fetcher = useCallback<BareFetcher>(
     async (resource, init) => {
       try {
         const response = await api.get(resource, init);
+        const data = (await response.json()) as ArbitraryObject;
```

Comment:
I added the dom.d.ts in PR #432 , after that PR is merged, you can use
```
const data = await response.json<ArbitraryObject>();
```

Reply:
updated